pages2/Pheno_Cluster_b.py

```diff
@@ -368,6 +377,9 @@ def main():
             st.number_input(label = "Number of markers to plot", 
                                 key = 'phenocluster__plot_diff_intensity_n_genes',
                                 step = 1)
+            st.toggle("Normalize total intensity", key='phenocluster__plot_normalize_total_intensity')
+            st.toggle("Log normalize", key='phenocluster__plot_log_normalize')
+            st.toggle("Z-score normalize columns", key='phenocluster__plot_zscore_normalize')
```

Comment:
Are these meant to be methods that could all be employed simultaneously and therefore st.toggle makes sense, or are you intending for only one of them to be applied at a time, in which case radio buttons would be more appropriate?

Looking at the code it seems you're fine with allowing any and all of them to be applied simultaneously as long as the order is correct, right? In that case everything looks fine, but I thought I'd check to make sure.